brainscore_vision/models/effnetb1_cutmixpatch_augmix_robust32_avge4e7_manylayers_324x288/requirements.txt

```diff
@@ -0,0 +1,5 @@
+numpy
+torch
+timm
+albumentations
+PIL
```

Comment:
@pellegreene PIL is no longer supported for Python 3 and instead has been replaced by [Pillow](https://pillow.readthedocs.io/en/stable/)